lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -14,6 +14,8 @@
 {-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE TypeOperators #-}
 
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
`BoundType` should be wrapped in `ApiT` to avoid the orphan instance. That pretty much the only reason we do use `ApiT` in the end :)